tests/settings.test.ts

```diff
@@ -36,9 +36,9 @@ test('handles server status updates correctly', async ({ page }) => {
 	await expect(page.getByLabel('Server')).toHaveValue('http://localhost:11434');
 
 	// The starting status is "connected"
-	await expect(page.getByText('disconnected')).not.toBeVisible();
-	await expect(page.getByText('connected', { exact: true })).toBeVisible();
-	await expect(page.getByText('connected', { exact: true })).toHaveClass(/badge--positive/);
+	await expect(page.getByText('Disconnected')).not.toBeVisible();
+	await expect(page.getByText('Connected')).toBeVisible();
+	await expect(page.getByText('Connected')).toHaveClass(/badge--positive/);
```

Comment:
Fixed it!

`Connected`/`Disconnected` were not capitalized in the i18n locales, but they showed as capitalized in the UI because they were transformed by CSS. Got rid of all of that and did other cleanup work in that component while I was there.